src/main/java/pro/sky/courseworktelegrambot/entities/StateButton.java

```diff
@@ -0,0 +1,44 @@
+package pro.sky.courseworktelegrambot.entities;
+
+import javax.persistence.*;
+
+@Entity
+@IdClass(StateButtonPK.class)
+public class StateButton {
+    @Id
+    private String state_id; //в работе не потребуется. Поле существует, только для hibernate
```

Comment:
```suggestion
    private String stateId; //в работе не потребуется. Поле существует, только для hibernate
```
camelCase, а не snake_case